uefi/efistub.txt

```diff
@@ -0,0 +1,87 @@
+EFISTUB
+________________________________________________________________________________
+
+For some UEFI users, the Linux kernel has the ability to act as an EFI
+executable, which simplifies the boot process by removing the need for a
+bootloader.
+
+
+Prerequisites
+________________________________________________________________________________
+
+Install efibootmgr
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ kiss b efibootmgr && kiss i efibootmgr                                   |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+After mounting efivarfs (@/uefi/efivarfs), enable EFISTUB support in the kernel
+with CONFIG_EFI_STUB=y. In menuconfig, this is located at:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   Processor type and features  --->                                          |
+|      [*] EFI runtime service support                                         |
+|      [*]   EFI stub support                                                  |
+|                                                                              |
```

Comment:
We should list the kernel options themselves. See https://k1ss.org/wiki/kernel/config